src/legacy/core_plugins/vis_type_timeseries/server/lib/get_fields.js

```diff
@@ -19,6 +19,7 @@
 import { SearchStrategiesRegister } from './search_strategies/search_strategies_register';
 import { uniq } from 'lodash';
 import { getIndexPatternObject } from './vis_data/helpers/get_index_pattern';
+import { isNestedField } from '../../../../../plugins/data/common';
```

Comment:
We probably should be re-exporting `isNestedField` from both `data/server/index.ts` and `data/public/index.ts` since the linter should complain about imports from `/common` once these items move to the new platform. 

Then the util can be imported from `/server` and `/public` instead.